packages/dendron-plugin-views/src/components/DendronTreeExplorerPanel.tsx

```diff
@@ -15,12 +15,15 @@ type OnExpandFunc = TreeProps["onExpand"];
 type OnSelectFunc = TreeProps["onSelect"];
 const DendronTreeExplorerPanel: DendronComponent = (props) => {
   const logger = createLogger("DendronTreeExplorerPanel");
-  const noteActive = props.ide.noteActive;
   const engine = props.engine;
   const { config, notes } = engine;
+  const numNotes = _.size(notes);
```

Comment:
since users can pass files as virtual notes, will this not also change based on that? maybe its not worth handling. if we decide not to handle it, can you add a `// OPTIMIZE:` comment as well as a backlog task for this?

speaking of which, can we add a new cookbook item in [[Views|dendron://dendron.docs/pkg.plugin-core.dev.cook#views]] for how to pass virtual notes into the view?

Reply:
> since users can pass files as virtual notes, will this not also change based on that?

I don't think so actually, the virtual notes aren't sent to the tree view, only note preview.

> can we add a new cookbook item

Got it!